hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/BackupDriver.java

```diff
@@ -120,6 +120,8 @@ private int parseAndRun(String[] args) throws IOException {
       type = BackupCommand.REPAIR;
     } else if (BackupCommand.MERGE.name().equalsIgnoreCase(cmd)) {
       type = BackupCommand.MERGE;
+    } else if (BackupCommand.CLEANUP.name().equalsIgnoreCase(cmd)) {
+      type = BackupCommand.CLEANUP;
     } else {
```

Comment:
I would name this something more specific, unless this command intends to clean up entries that may be left behind for full and incremental backups as well